src/types.ts

```diff
@@ -31,7 +31,7 @@ export enum productTypes {
 
 export type Product = {
   id: number
-  serialNumber: number
```

Comment:
??😂

Reply:
I swapped to serialnumber (with small letter) for react warning fix